src/System.Private.Uri/src/System.Private.Uri.csproj

```diff
@@ -12,10 +12,17 @@
     <DefineConstants>$(DefineConstants);INTERNAL_GLOBALIZATION_EXTENSIONS</DefineConstants>
     <!-- Suppress warnings for type conflicts between SafeFileHandle in partial facade and mscorlib -->
     <NoWarn>0436</NoWarn>
-    <PackageTargetFramework Condition="'$(PackageTargetFramework)'==''">netstandard1.0</PackageTargetFramework>
-    <SkipCommonResourcesIncludes Condition="'$(TargetGroup)'=='netcore50aot' Or '$(TargetGroup)'=='netstandard13aot'">true</SkipCommonResourcesIncludes>
+    <SkipCommonResourcesIncludes Condition="'$(TargetGroup)'=='netcore50aot' Or '$(TargetGroup)'=='netstandard13aot' Or '$(TargetGroup)'=='uap101aot'">true</SkipCommonResourcesIncludes>
     <NuGetTargetMoniker Condition="'$(TargetGroup)' == ''">.NETStandard,Version=v1.0</NuGetTargetMoniker>
   </PropertyGroup>
+  <ItemGroup Condition="'$(PackageTargetFramework)' == ''">
+    <PackageDestination Include="lib/netstandard1.0">
+      <TargetFramework>netstandard1.0</TargetFramework>
+    </PackageDestination>
+    <PackageDestination Include="lib/uap10.1">
+      <TargetFramework>uap10.1</TargetFramework>
+    </PackageDestination>
+  </ItemGroup>
```

Comment:
Multiple values in PackageTargetFramework is not supported. That's why I removed PackageTargetFramework and expressed it as PackageDestination items. 
I tried this again per your comment after pulling changes post merge from master but it's still the same. FWIW, I can't see any example of multi values PackageTargetFramework in CoreFX repo.